server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -56,7 +56,7 @@
 import io.crate.testing.UseRandomizedSchema;
 import io.netty.handler.codec.http.HttpResponseStatus;
 
-@IntegTestCase.ClusterScope()
+@IntegTestCase.ClusterScope(scope = IntegTestCase.Scope.TEST)
```

Comment:
this is needed to reset OID after test run otherwise OID assertions are non-deterministic.